crates/forge/tests/fixtures/zk/DeployMultisig.s.sol

```diff
@@ -0,0 +1,88 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.17;
+
+import "forge-std/Script.sol";
+import "zksync-contracts/zksync-contracts/l2/system-contracts/libraries/SystemContractsCaller.sol";
+import "@openzeppelin/contracts/utils/cryptography/ECDSA.sol";
+import "../src/AAFactory.sol";
+import "../src/TwoUserMultisig.sol";
+
+contract DeployMultisig is Script {
+    function run() external {
+        // Owners for the multisig account
+        // Can be random
+        address owner1 = makeAddr("OWNER_1");
+        address owner2 = makeAddr("OWNER_2");
+
+        // Read artifact file and get the bytecode hash
+        string memory artifact = vm.readFile("zkout/TwoUserMultisig.sol/TwoUserMultisig.json");
+        bytes32 multisigBytecodeHash = vm.parseJsonBytes32(artifact, ".hash");
+        console.log("Bytecode hash: ");
+        console.logBytes32(multisigBytecodeHash);
+        bytes32 salt = "CEACREACREA";
+
+        vm.startBroadcast();
+        AAFactory factory = new AAFactory(multisigBytecodeHash);
+        console.log("Factory deployed at: ", address(factory));
+        (bool _success,) = address(vm).call(abi.encodeWithSignature("zkUseFactoryDep(string)", "TwoUserMultisig"));
+        require(_success, "Deployment failed");
+        factory.deployAccount(salt, owner1, owner2);
+        string memory factoryArtifact = vm.readFile("zkout/AAFactory.sol/AAFactory.json");
+        bytes32 factoryBytecodeHash = vm.parseJsonBytes32(factoryArtifact, ".hash");
+        Create2Factory create2Factory = new Create2Factory();
```

Comment:
I thought we'd use the `AAFactory` we deployed above or use this new one to deploy the new account...
Or are we checking a new instance?